spicy/toolchain/src/compiler/validator.cc

```diff
@@ -36,8 +38,8 @@ bool supportsLiterals(QualifiedType* t) {
 // Helper to make sure a field's attributes are consistent. This is type-independent.
 hilti::Result<hilti::Nothing> checkFieldAttributes(type::unit::item::Field* f) {
     // Can't combine ipv4 and ipv6
-    auto v4 = f->attributes()->find(hilti::Attribute::Kind::IPv4);
-    auto v6 = f->attributes()->find(hilti::Attribute::Kind::IPv6);
+    bool v4 = f->attributes()->has(hilti::Attribute::Kind::IPv4);
+    bool v6 = f->attributes()->has(hilti::Attribute::Kind::IPv6);
```

Comment:
Personally, I'd stay with `auto` here but matter of preference.

EDIT: Below it still uses `auto` with `has()`.